components/plurals/src/lib.rs

```diff
@@ -147,6 +150,7 @@ pub enum PluralRuleType {
 #[derive(Debug, PartialEq, Eq, Clone, Copy, Hash, Ord, PartialOrd)]
 #[cfg_attr(feature = "datagen", derive(serde::Serialize))]
 #[cfg_attr(feature = "serialize", derive(serde::Deserialize))]
+#[allow(clippy::exhaustive_enums)] // this type is stable
```

Comment:
I assume this will never change? @zbraniecki

Reply:
This is an example enum that we've discussed previously in this context, as one that is in the gray area. We agreed that this enum changes so rarely that we can make it stable, and save potential changes for major version bumps.